index.html

```diff
@@ -353,6 +354,69 @@ <h2>Ballerina in Action</h2>
                   </div>
                </div>
             </div>
+            <div id="fifthTab" class="tab-pane fade"  >
+            <div class="row cdisplayFlex" style="background-color: #f5f5f5;">
+                <div class="col-lg-12 col-md-12 col-sm-12 focusPane" style="padding:20px;border: 1px #c7c7c7 solid;">
+                    <div class="FullCode" style="background-color: #f5f5f5;">
+                        <div class="highlight">
+                            <pre class="language-ballerina basic ballerina">
+                                <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerinax/kafka;
+import ballerina/lang.value;
+
+configurable string groupId = "group-id";
+configurable string listeningTopic = "topic-orders";
+configurable string publishingTopic = "topic-success-orders";
+configurable decimal pollingInterval = 1;
+configurable string kafkaEndpoint = kafka:DEFAULT_URL;
+
+type Order readonly & record {|
+    int id;
+    string name;
+    PaymentStatus status;
+|};
+
+enum PaymentStatus {
+    SUCCESS,
+    FAIL
+}
+
+final kafka:ConsumerConfiguration consumerConfigs = {
+    groupId: groupId,
+    topics: [listeningTopic],
+    offsetReset: kafka:OFFSET_RESET_EARLIEST,
+    pollingInterval: pollingInterval
+};
+
+service on new kafka:Listener(kafkaEndpoint, consumerConfigs) {
+
+    private kafka:Producer kafkaProducer;
+
+    function init() returns error? {
+        self.kafkaProducer = check new (kafkaEndpoint);
+    }
+
+    remote function onConsumerRecord(kafka:ConsumerRecord[] records) returns error? {
+        check from var {value} in records
+            let var orderString = check string:fromBytes(value)
+            let Order 'order = check value:fromJsonStringWithType(orderString, Order)
```

Comment:
```suggestion
            let Order 'order = check value:fromJsonStringWithType(orderString)
```

We don't need the parameter `Order` here. It should be inferred from the LHS.

Reply:
Added in 76f775c